Admin/BaseMediaAdmin.php

```diff
@@ -50,6 +53,15 @@ public function __construct($code, $class, $baseControllerName, Pool $pool, Cate
     /**
      * {@inheritdoc}
      */
+    public function configureRoutes(RouteCollection $collection)
+    {
+        $collection->add('multi_upload', 'upload/multi');
+        $collection->add('multi_upload_ajax', 'upload/multi/ajax/{provider}');
```

Comment:
Also, putting "ajax" in the route and the url feels weird. I mean you don't put "post" or "get", so why would you put ajax? Please focus on the purpose, not on the technology.